app/helpers/emails_helper.rb

```diff
@@ -31,8 +31,8 @@ def action_title(url)
   end
 
   def color_email_diff(diffcontent)
-    formatter = Rugments::Formatters::HTML.new(cssclass: "highlight", inline_theme: :github)
-    lexer = Rugments::Lexers::Diff.new
+    formatter = Rouge::Formatters::HTML.new(cssclass: "highlight", inline_theme: :github)
```

Comment:
It looks like `html_inline.rb` doesn't expect a string as the theme. As a workaround, this seems to make things work:

```ruby
    formatter = Rouge::Formatters::HTML.new(cssclass: "highlight", inline_theme: Rouge::Themes::Github.new)
```

But I'm confused because while I find `html_inline.rb` in the .gem file, it doesn't appear to be in the [source](https://github.com/jneen/rouge/tree/master/lib/rouge/formatters). Is this auto-generated, @jneen?